static/js/components/ShowClassification.jsx

```diff
@@ -1,22 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
 import PropTypes from "prop-types";
 import Tooltip from "@mui/material/Tooltip";
 import Chip from "@mui/material/Chip";
-
+import DeleteIcon from "@mui/icons-material/Delete";
 import makeStyles from "@mui/styles/makeStyles";
 
+import { showNotification } from "baselayer/components/Notifications";
+import Button from "./Button";
+import ConfirmDeletionDialog from "./ConfirmDeletionDialog";
+
+import * as sourceActions from "../ducks/source";
+
 export const useStyles = makeStyles((theme) => ({
   chip: {
     margin: theme.spacing(0.5),
     fontSize: "1.2rem",
     fontWeight: "bold",
   },
+  classificationDelete: {
+    cursor: "pointer",
+    fontSize: "2em",
+    position: "absolute",
+    padding: 0,
+    right: 0,
+    top: 0,
+  },
+  classificationDeleteDisabled: {
+    opacity: 0,
+  },
 }));
 
 const ClassificationRow = ({ classifications }) => {
   const classes = useStyles();
+  const dispatch = useDispatch();
+
+  const currentUser = useSelector((state) => state.profile);
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const [classificationToDelete, setClassificationToDelete] = useState(null);
+  const openDialog = (id) => {
+    setDialogOpen(true);
+    setClassificationToDelete(id);
+  };
+  const closeDialog = () => {
+    setDialogOpen(false);
+    setClassificationToDelete(null);
+  };
+
+  const deleteClassification = () => {
+    dispatch(sourceActions.deleteClassification(classificationToDelete)).then(
+      (result) => {
+        if (result.status === "success") {
+          dispatch(showNotification("Classification deleted"));
+          closeDialog();
+        }
+      }
+    );
+  };
 
   const classification = classifications[0];
+
+  const permission =
+    currentUser.permissions.includes("System admin") ||
+    currentUser.permissions.includes("Manage groups") ||
```

Comment:
Why "Manage groups"?

Reply:
@guynir42 That's just the permission we have on the classifications (I expect this is as close as we get to "person who manages source stuff so can manage classifications")